tests/console/bind.pm

```diff
@@ -7,6 +7,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package: libopenssl-devel libmysqlclient-devel bind rpm-build perl-IO-Socket-INET6
+# bind rpm-build bind-utils net-tools-deprecated perl-IO-Socket-INET6 perl-Socket6 perl-Net-DNS python3-dnspython
```

Comment:
Are we sure that [openqa-summary-tests](https://gitlab.suse.de/qa-maintenance/openqa-summary-tests) parses the second line as well?

Reply:
According to Marcelo, it parses the text until it finds the next tag.